docs/commercial/custom_domains.rst

```diff
@@ -1,12 +1,16 @@
 Custom Domains
 ==============
 
+.. note:: These directions are for projects hosted on Read the Docs for Business.
+          For setting up a custom domain on a project hosted on readthedocs.org,
+          please read our :doc:`community documentation </custom_domains>`.
```

Comment:
Instead of a separate page for these docs, the feature we discussed adding with this PR is to use `sphinx_tabs` extension to make it really easy to click between .org and .com directions for areas where docs deviated. This seems like a more digestible pattern for the reader. Do we want to address that now?

Reply:
I think that's a separate issue. This PR is purely marketing as opposed to actually editing much in the way of documentation and merging two docs into one.